qiskit_experiments/framework/base_experiment.py

```diff
@@ -264,7 +266,31 @@ def run(
         # Generate and transpile circuits
         transpile_opts = copy.copy(experiment.transpile_options.__dict__)
         transpile_opts["initial_layout"] = list(experiment.physical_qubits)
-        circuits = transpile(experiment.circuits(), experiment.backend, **transpile_opts)
+
+        circuits = experiment.circuits()
+
+        # Scheduling parameters
+        if (
+            experiment.backend
+            and not experiment.backend.configuration().simulator
+            and not isinstance(experiment.backend, FakeBackend)
+        ):
+            updated_schedule = False
+            for circ in circuits:
+                for op, _, _ in circ.data:
+                    if op.name == "delay":
+                        timing_constraints = transpile_opts.get("timing_constraints", {})
+                        if "acquire_alignment" not in timing_constraints:
+                            timing_constraints["acquire_alignment"] = 16
```

Comment:
This is no longer necessary because all backends have been updated to report constraint values now. Backend that doesn't report the value is considered to have no constraint. If you hard-code the constant value at experiment level, you may trigger non-necessary alignment.